pkg/object/graph/graph.go

```diff
@@ -162,3 +162,34 @@ func (cde CyclicDependencyError) Error() string {
 	}
 	return errorBuf.String()
 }
+
+// EdgeSet is a list wrapper that allows set equality comparison.
```

Comment:
The EdgeSet code is unnecessary. If you want to advocate for this minor change, please do it in another PR.

Reply:
removed. I thought this would be a nice clean way to more efficiently compare edges using sorting, but it was just a nice to have. I've removed it to reduce the size of this PR.